banzai_floyds/dbs.py

```diff
@@ -169,3 +175,57 @@ def add_order_location(db_address, instrument_id, xdomainmin, xdomainmax,
                                      good_after=good_after, good_until=good_until)
         db_session.add(new_location)
         db_session.commit()
+
+
+def get_cal_record(image, calibration_type, selection_criteria, db_address):
```

Comment:
Can we have a short docstring here describing the logic of what this function does?
Just imagining the day in a few months were someone asks me how Banzai finds a calibration and I have to go parse code instead of just reading it here...

Reply:
Basically just a copy/paste of the note in the docs should be good.

Reply:
Done.